service/service.go

```diff
@@ -0,0 +1,125 @@
+package service
+
+import (
+	"context"
+
+	"github.com/ONSdigital/dp-legacy-cache-proxy/config"
+	"github.com/ONSdigital/dp-legacy-cache-proxy/proxy"
+	"github.com/ONSdigital/log.go/v2/log"
+	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
+	"go.opentelemetry.io/contrib/instrumentation/github.com/gorilla/mux/otelmux"
+	// "go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
```

Comment:
Well spotted, but this is intentional. It's here just in case we need it in line 35. If it turns out to be unneeded, we can remove this as part of DIS-392.